app/Website/src/Http/Controllers/BlogPostController.php

```diff
@@ -30,4 +31,17 @@ public function index(Request $request): AnonymousResourceCollection
 
         return BlogPostsResource::collection($blogs);
     }
+
+    #[Endpoint('Website/Get Blog Post', <<<'DESC'
+  This endpoint retrieve blogpost by a slug.
+DESC)]
+    public function show(string $slug): SingleBlogPostResource
+    {
+        $blog = BlogPost::with('author', 'tags')
+            ->where('status', BlogPostStatus::PUBLISHED)
+            ->where('slug', 'like', "%{$slug}%")
```

Comment:
Slugs are supposed to be filtered with a 100% match so "like" wouldn't work.

It should be a "where equals" query.